app/controllers/coronavirus_form/able_to_leave_controller.rb

```diff
@@ -20,12 +20,24 @@ def submit
       render controller_path
     else
       update_session_store
+      write_responses
       redirect_to results_path
     end
   end
 
 private
 
+  # We're using this method to reduce the precision of timekeeping so that
+  # responses cannot be correlated with analytics data
+  def time_hour_floor
+    time = Time.current
+    time - time.sec - time.min % 60 * 60
+  end
+
+  def write_responses
+    FormResponse.create({ form_response: session, created_at: time_hour_floor })
```

Comment:
I'm surprised linter didn't complain about the curly braces 🤔

Reply:
me too, have changed